src/Format/Format.GeoJSON.js

```diff
@@ -28,6 +28,6 @@ L.Format.GeoJSON = L.Format.Base.extend({
   },
 
   generateLayer: function (feature) {
-    return L.GeoJSON.geometryToLayer(feature, this.options.pointToLayer || null, this.options.coordsToLatLng || null, null);
+    return L.GeoJSON.geometryToLayer(feature, this.options || null, this.options.coordsToLatLng || null, null);
```

Comment:
[geometryToLayer ](http://leafletjs.com/reference-1.0.2.html#geojson-geometrytolayer) have 2 arguments, I don't remeber for which purpose here were be 4 :), could you also remove unnecessary args?